app/models/bus_stop.rb

```diff
@@ -48,8 +48,22 @@ def route_list
     routes.pluck(:number).sort.join(', ')
   end
 
-  def self.to_csv(limited_attributes: false)
-    # TODO
+  def self.to_csv
+    CSV.generate headers: true do |csv|
+      all_field_names = Field.pluck(:name)
```

Comment:
So what about data that we have (BSFs that exist), but for which there aren't matching fields anymore? Is it fine that those don't end up in the CSV?

Reply:
Are you saying that I should add `dependent: :destroy` to the `fields` model?

Reply:
No, definitely not that. Quite the opposite: do we want to include those data points in the CSV?

Reply:
Oh, got it.

Reply:
Yes

Reply:
I'm a bit confused on how to design this. So, if I have a BSF that exists but there isn't a matching field, then how do I name the columns of the CSV? Will the BSF still have a `field_name` or will that be `nil`?

Reply:
Ok, thanks @dfaulken